### PR TITLE
Javascript classes

### PR DESCRIPTION
**Pull request by MustakimMasum**
Activities of the branch-

- I have added a JavaScript folder with index.js and class.js files
- I have used JavaScript classes for the project in the class.js file
- I have also added more styling in the style.css file to set the background of the project.

I kindly ask for a review.

### CODE REVIEW

**Comment by MustakimMasum** [comment]
We have been facing linter errors showing us version conflict in the upstream. We tried to fix the linter errors for the past two days. However, failing to resolve the version conflicts, we created a new repo and populated our codes into the new repo.

File: idex.html
Comment (shella12) [inline idex.html:1]: - [ ] [OPTIONAL] The file name `idex.html` is misspelled. The correct file name is `index.html`. This is the default file that is loaded when a user visits your website, so I would highly recommend you to fix the file name.

File: idex.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            <div class="books-list"></div>
+            <hr>
+            <h1>Add a new book</h1>
+            <form class="add-form">
+                <label for="title" hidden>Title:</label>
+                <input type="text" class="title" required placeholder="Title"/>
+                <label for="author" hidden>Author:</label>
+                <input type="text" class="author" required placeholder="Author"/>
+                <button type="submit" class="form-btn">Add</button>
+            </form>
```
Comment (shella12) [inline idex.html]: - [ ] [OPTIONAL] I noticed that you're using the `for` attribute on the labels, but you're not using the `name` attribute on the input elements. Without the name attribute label is not referencing to anything so it's just confusing the browser. Also

The `name` attribute is important for two reasons:

1. It allows you to reference the input elements in JavaScript.
2. It allows you to submit the values of the input elements when the form is submitted. Without the name attribute, the values of the input elements will not be submitted when the form is submitted.

I recommend that you add the` name` attribute to the input elements. For example, you could change the code to the following:

```html

<form class="add-form">
<label for="title">Title:</label>
<input type="text" class="title" required placeholder="Title" name="title"/>
<label for="author">Author:</label>
<input type="text" class="author" required placeholder="Author" name="author"/>
<button type="submit" class="form-btn">Add</button>
</form>

```

**Review by shella12 — APPROVED** [review]
Hi @firepicaso and @albertkantwi, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- used JavaScript` class` instead of array and object✔️ 
- Add book function is working✔️ 
- Remove book function is working ✔️ 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**